src/sentry/templates/sentry/emails/feedback.html

```diff
@@ -0,0 +1,243 @@
+{% extends "sentry/emails/base.html" %}
+
+{% load timezone from tz %}
+{% load sentry_avatars %}
+{% load sentry_helpers %}
+{% load sentry_features %}
+{% load sentry_assets %}
+{% load i18n static %}
+
+{% block head %}
+  {{ block.super }}
+  <style type="text/css" inline="false">
+    @media only screen and (max-device-width: 480px) {
+      /* Hide CTA in header on mobile */
+      .header-buttons .integration-link {
+        display: none !important;
+      }
+      .banner {
+        display: none !important;
+      }
+    }
+    @media only screen and (max-device-width: 768px) {
+      .text-desktop-only {
+        display: none !important;
+      }
+    }
+  </style>
+{% endblock %}
+
+{% block preheader %}
+  {{ group_header }} from {{ project_label }}.
+{% endblock %}
+
+{% block header %}
+  <div class="header-with-buttons">
+    {{ block.super }}
+    <div class="header-buttons">
+      {% if not has_alert_integration %}
+        <a href="{{ slack_link }}" class="btn btn-default integration-link">
+          <img src="{% absolute_asset_url 'sentry' 'images/email/slack-logo.png' %}" class="logo-small" />
+          Set up in Slack
+        </a>
+      {% endif %}
+      <a href="{{ link }}" class="btn view-on-sentry">View on Sentry</a>
+    </div>
+  </div>
+{% endblock %}
+
+{% block content %}
+
+<div class="container">
+  <div class="inner">
+    <h2 {% if notification_reason %}style="margin-bottom: 4px"{% else %}style="margin-bottom: 15px"{% endif %}>
+        {% if has_issue_states and group_header %}
+        {{ group_header }}
+        {% else %}
+        New Feedback from <a href="{{group.project.get_absolute_url}}">{{ project_label }}</a>
+        {% if environment %} in {{ environment }}{% endif %}
+        {% endif %}
+    </h2>
+    {% if notification_reason %}
+      <div class="event-notification-reason">
+        {{ notification_reason }}
+      </div>
+    {% endif %}
+
+    {% if enhanced_privacy %}
+      <div class="event">
+        <div class="event-id">ID: {{ event.event_id }}</div>
+          {% if timezone %}
+            <div class="event-date">{{ event.datetime|timezone:timezone|date:"N j, Y, g:i:s a e"}}</div>
+          {% else %}
+            <div class="event-date">{{ event.datetime|date:"N j, Y, g:i:s a e"}}</div>
+          {% endif %}
+      </div>
+
+      <div class="notice">Details about this issue are not shown in this notification since enhanced privacy
+        controls are enabled. For more details about this issue, <a href="{{ link }}">view this issue on Sentry</a>.</div>
```

Comment:
```suggestion
      <div class="notice">Details about this feedback are not shown in this notification since enhanced privacy
        controls are enabled. For more details about this feedback, <a href="{{ link }}">view this issue on Sentry</a>.</div>
```